2021-11-22 01:21:27,286 INFO    MainThread:125399 [wandb_setup.py:_flush():71] setting env: {'project': '11_21train'}
2021-11-22 01:21:27,286 INFO    MainThread:125399 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-22 01:21:27,287 INFO    MainThread:125399 [wandb_init.py:_log_setup():371] Logging user logs to /home/yu-jw19/venom/project2/wandb/run-20211122_012127-1xn3rnbz/logs/debug.log
2021-11-22 01:21:27,287 INFO    MainThread:125399 [wandb_init.py:_log_setup():372] Logging internal logs to /home/yu-jw19/venom/project2/wandb/run-20211122_012127-1xn3rnbz/logs/debug-internal.log
2021-11-22 01:21:27,287 INFO    MainThread:125399 [wandb_init.py:init():404] calling init triggers
2021-11-22 01:21:27,287 INFO    MainThread:125399 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2021-11-22 01:21:27,288 INFO    MainThread:125399 [wandb_init.py:init():449] starting backend
2021-11-22 01:21:27,288 INFO    MainThread:125399 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-22 01:21:27,307 INFO    MainThread:125399 [backend.py:ensure_launched():199] starting backend process...
2021-11-22 01:21:27,317 INFO    MainThread:125399 [backend.py:ensure_launched():204] started backend process with pid: 125533
2021-11-22 01:21:27,318 INFO    MainThread:125399 [wandb_init.py:init():458] backend started and connected
2021-11-22 01:21:27,320 INFO    MainThread:125399 [wandb_init.py:init():519] updated telemetry
2021-11-22 01:21:27,342 INFO    MainThread:125399 [wandb_init.py:init():549] communicating current version
2021-11-22 01:21:28,492 INFO    MainThread:125399 [wandb_init.py:init():554] got version response 
2021-11-22 01:21:28,492 INFO    MainThread:125399 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-11-22 01:21:29,009 INFO    MainThread:125399 [wandb_init.py:init():592] starting run threads in backend
2021-11-22 01:21:29,056 INFO    MainThread:125399 [wandb_run.py:_console_start():1816] atexit reg
2021-11-22 01:21:29,057 INFO    MainThread:125399 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2021-11-22 01:21:29,057 INFO    MainThread:125399 [wandb_run.py:_redirect():1695] Redirecting console.
2021-11-22 01:21:29,060 INFO    MainThread:125399 [wandb_run.py:_redirect():1751] Redirects installed.
2021-11-22 01:21:29,060 INFO    MainThread:125399 [wandb_init.py:init():619] run started, returning control to user process
2021-11-22 01:21:29,068 INFO    MainThread:125399 [wandb_run.py:_config_callback():962] config_cb None None {'dataset': 'emotiondataset', 'name': 'firsttest_network=resnet,layers=44', 'no': '0', 'model': 'resnet', 'print_freq': 10, 'layers': 44, 'droprate': 0.3, 'cardinality': 8, 'growth_rate': 12, 'compression_rate': 0.5, 'bn_size': 4, 'alpha': 200, 'N': 2, 'M': 10, 'augment': True, 'randaugment': True, 'checkpoint': 'checkpoint', 'resume': False, 'cos_lr': True, 'initial_learning_rate': 0.1, 'changing_lr': [80, 120], 'en_wandb': True, 'warm_up': False, 'exp_id': '00037_network=resnet,layers=44', 'commit': 'not_set'}
2021-11-22 01:21:58,260 INFO    MainThread:125399 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-11-22 01:21:58,261 INFO    MainThread:125399 [wandb_run.py:_restore():1758] restore
2021-11-22 01:22:00,599 INFO    MainThread:125399 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 970
}

2021-11-22 01:22:00,721 INFO    MainThread:125399 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 970
}

2021-11-22 01:22:01,222 INFO    MainThread:125399 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 970
}

2021-11-22 01:22:02,055 INFO    MainThread:125399 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 8995
}

2021-11-22 01:22:02,159 INFO    MainThread:125399 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 9022
}

2021-11-22 01:22:02,262 INFO    MainThread:125399 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 9022
}

2021-11-22 01:22:02,365 INFO    MainThread:125399 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 9022
}

2021-11-22 01:22:02,468 INFO    MainThread:125399 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 9022
}

2021-11-22 01:22:02,572 INFO    MainThread:125399 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 9022
}

2021-11-22 01:22:02,675 INFO    MainThread:125399 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6766
  total_bytes: 9022
}

2021-11-22 01:22:02,779 INFO    MainThread:125399 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9022
  total_bytes: 9022
}

2021-11-22 01:22:02,882 INFO    MainThread:125399 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9022
  total_bytes: 9022
}

2021-11-22 01:22:02,985 INFO    MainThread:125399 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9022
  total_bytes: 9022
}

2021-11-22 01:22:03,088 INFO    MainThread:125399 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9022
  total_bytes: 9022
}

2021-11-22 01:22:03,191 INFO    MainThread:125399 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9022
  total_bytes: 9022
}

2021-11-22 01:22:03,294 INFO    MainThread:125399 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9022
  total_bytes: 9022
}

2021-11-22 01:22:03,609 INFO    MainThread:125399 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9022
  total_bytes: 9022
}

2021-11-22 01:22:04,670 INFO    MainThread:125399 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9022
  total_bytes: 9022
}
local_info {
}

2021-11-22 01:22:05,760 INFO    MainThread:125399 [wandb_run.py:_append_files():2184] logging synced files
